### PR TITLE
Add PRINSEQPLUSPLUS

### LINKED ISSUE #1480: new module: PRINSEQ-PLUS-PLUS
I think it would be good to have a module for [PRINSEQ-PLUS-PLUS](https://bioconda.github.io/recipes/prinseq-plus-plus/README.html)

- [x] This module does not exist yet with the [`nf-core modules list`](https://github.com/nf-core/tools#list-modules) command
- [x] There is no [open pull request](https://github.com/nf-core/modules/pulls) for this module
- [x] There is no [open issue](https://github.com/nf-core/modules/issues) for this module
- [x] If I'm planning to work on this module, I added myself to the `Assignees` to facilitate tracking who is working on the module

### PR DESCRIPTION
**Pull request by jfy133**
## PR checklist

Closes #1480 

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [x] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [x] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [x] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: tests/modules/prinseqplusplus/test.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  command: nextflow run tests/modules/prinseqplusplus -entry test_prinseqplusplus_single_end -c tests/config/nextflow.config
+  tags:
+    - prinseqplusplus
+  files:
+    - path: output/prinseqplusplus/test.log
+      contains:
+        - "reads removed by -lc_entropy"
+    - path: output/prinseqplusplus/test_bad_out.fastq.gz
+    - path: output/prinseqplusplus/test_good_out.fastq.gz
+    - path: output/prinseqplusplus/versions.yml
```
Comment (FriederikeHanssen) [inline tests/modules/prinseqplusplus/test.yml:11]: I think we said we wouldn't check the md5sum anymore, too much hassle for version updates https://github.com/nf-core/modules/pull/1459/commits/b90e590dff5d96c382ac2c50cc47f8f072ace9dd

**Review by FriederikeHanssen — APPROVED** [review]

File: tests/modules/prinseqplusplus/test.yml (hunk 1 above)
Comment (jfy133) [reply tests/modules/prinseqplusplus/test.yml:11]: Is that documented somewhere and I missed it

Comment (FriederikeHanssen) [reply tests/modules/prinseqplusplus/test.yml:11]: Can't dinf it. @maxulysse , I saw it in your PR, any docs on this?

File: tests/modules/prinseqplusplus/test.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      md5sum: e194788e64fd36a8ac9d4ee62ede0b40
```
Comment (jfy133) [inline tests/modules/prinseqplusplus/test.yml:12]:
```suggestion
```

File: tests/modules/prinseqplusplus/test.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      contains:
+        - "reads removed by -lc_entropy"
+    - path: output/prinseqplusplus/test_bad_out_R1.fastq.gz
+    - path: output/prinseqplusplus/test_bad_out_R2.fastq.gz
+    - path: output/prinseqplusplus/test_good_out_R1.fastq.gz
+    - path: output/prinseqplusplus/test_good_out_R2.fastq.gz
+    - path: output/prinseqplusplus/test_single_out_R1.fastq.gz
+    - path: output/prinseqplusplus/test_single_out_R2.fastq.gz
+    - path: output/prinseqplusplus/versions.yml
+      md5sum: b3b27b17128e36a3bdd9cfbcc058d8bc
```
Comment (jfy133) [inline tests/modules/prinseqplusplus/test.yml:29]:
```suggestion
```

**Pull request merged**